### ISSUE TITLE
Add ability to get whole cookies within browser

### ISSUE THREAD

**Issue by DenisBalan** [hardkoded/puppeteer-sharp#1526]
Need feature to get all cookies (not from page.GetCookiesAsync which is returning current page cookies)
The need for this is that you dont know all url's to get cookies for them,
More than that Chrome DevTools Protocol permit that, but pupetter API's doesnt.
```
...

await page.GoToAsync("https://google.com/");
await page.GoToAsync("https://yandex.ru/");

var resp = await page.GetCookiesAsync(new string[0]); // returning cookies for last Url
// based on https://github.com/hardkoded/puppeteer-sharp/blob/master/lib/PuppeteerSharp/Page.cs#L629

// need to get all cookies?
// NetworkGetCookiesResponse is internal
// so i have to re-create it in the code
var resp2 = await page.Client.SendAsync<NetworkGetCookiesResponse>("Network.getCookies");
```

tldr; need ability to get all browser cookies from pupetter API without specifying concrete urls

**Comment by kblok** [comment]
Closed due to inactivity. Feel free to reopen it if needed.
